pkgs/development/python-modules/pyscf/default.nix

```diff
@@ -13,6 +13,10 @@ buildPythonPackage rec {
     sha256  = "0xbwkjxxysfpqz72qn6n4a0zr2h6sprbcal8j7kzymh7swjy117w";
   };
 
+  # Backport from the 2.0.0 alpha releases of PySCF.
+  # H5Py > 3.3 deprecates the file modes, that PySCF sets.
+  patches = [ ./h5py.patch ];
```

Comment:
Is there an upstream commit that fixes the problem?

Reply:
Yes, there is [this one](https://github.com/pyscf/pyscf/commit/967f7cf39ab8d273fb6bae6d43b1609c14ef4b39). But it is a merge commit and touches many other files. I haven't found anything how to get a patch from GitHub for just one file from a larger commit diff.

Reply:
I have no idea either how to pick that.

Reply:
Alright, then I guess we keep it for the moment as it is. When 2.0.0 becomes stable, the problem solves itself.